x11-wm/bspwm/bspwm-0.9.12.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit toolchain-funcs
+
+DESCRIPTION="Tiling window manager based on binary space partitioning"
+HOMEPAGE="https://github.com/baskerville/bspwm/"
+SRC_URI="https://github.com/baskerville/bspwm/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~amd64 ~riscv ~x86"
+IUSE="examples"
+
+DEPEND="
+	x11-libs/libxcb
+	x11-libs/xcb-util
+	x11-libs/xcb-util-wm
+"
+RDEPEND="${DEPEND}
+	x11-misc/sxhkd
+"
+
+src_compile() {
+	emake CC="$(tc-getCC)"
+}
+
+src_install() {
+	emake DESTDIR="${D}" PREFIX="${EPREFIX}/usr" DOCPREFIX="${EPREFIX}/usr/share/doc/${PF}" install
+
+	exeinto /etc/X11/Sessions
+	newexe "${FILESDIR}"/${PN}-session ${PN}
+
+	insinto /usr/share/xsessions
+	doins contrib/freedesktop/bspwm.desktop
+
+	insinto /etc/xdg/sxhkd
+	doins examples/sxhkdrc
+
+	if use examples ; then
+		dodoc -r examples
+		docompress -x /usr/share/doc/${PF}/examples
+	fi
```

Comment:
Now this is the reason I didn't merge this already and update EAPI by myself. This package install size is so small, even with `+examples` use flag, I don't know if having this flag makes any sense? Would be great to just install these files unconditionally every time, since they seem to be a lot of help too. (remember to remove the `IUSE` line if you agree)

Let me know if you agree/disagree. I got my eyes on this now so if you force-push an updated ebuild, with newer EAPI and "examples" use flag removed, I should spot it relatively fast.